vendor/aftereffects/OpenColorIO_AE_Context.cpp

```diff
@@ -656,8 +656,8 @@ void OpenColorIO_AE_Context::setupLUT(OCIO_Invert invert, OCIO_Interp interpolat
     
     if(invert == OCIO_INVERT_EXACT)
     {
-        _cpu_processor = _processor->getOptimizedCPUProcessor(OCIO::OPTIMIZATION_DEFAULT, OCIO::FINALIZATION_EXACT);
-        _gpu_processor = _processor->getOptimizedGPUProcessor(OCIO::OPTIMIZATION_DEFAULT, OCIO::FINALIZATION_EXACT);
+        _cpu_processor = _processor->getOptimizedCPUProcessor(OCIO::OPTIMIZATION_LOSSLESS);
```

Comment:
Hey Doug, with regards to this specific comment, I do use the default optimization in general, but this is a little feature I added where the user can use LOSSLESS (i.e. EXACT) if they want to. It's just an option, and a fairly well-hidden one at that.

But I would be happy to have a chat with you about other OCIO v2 features. I did experiment a little with categories but I don't fully understand them and don't have any v2 configurations to test with. Maybe you could educate me. I tried the integer buffers at one point and found they were actually slower. Maybe I should try them again.

Reply:
It's totally fine to allow different optimization levels.  Sorry, I didn't realize you had done that.

Given that Photoshop is often starting with integer pixel buffers, you should be getting a big performance boost for many color space conversions.  For example, in the ACES config, doing a ColorSpaceTransform where the source space is one of the log color spaces that starts with an inverse Lut1D, it should be a lot faster.  But you would need to be VERY_GOOD rather than LOSSLESS to get the speed-up.  There is also a stand-alone tool called ocioperf that may be useful when looking at performance issues.

There is a v2 demo config in the documentation now, so you could take a look at that.  We'd be happy to chat further at any time, DM me on the OCIO Slack when you're ready and we could set up a time.